package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Sia-UI",
-  "version": "v1.0.2",
+  "version": "v1.0.0",
```

Comment:
I had manually downgraded the string to verify that the update component was showing given an outdated UI version string and accidentally committed the downgraded string